### PR TITLE
[Tables] fixing README samples in tables

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/data/aztables/README.md [hunk 1]
```diff
@@ -84,39 +123,74 @@ connection string to the client's `NewServiceClientFromConnectionString` method.
...
+func main() {
+    connStr := "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
+    svc, err := NewServiceClientFromConnectionString(connStr, nil)
+    if err != nil {
+        panic(err)
+    }
 
-client, err := svc.CreateTable(context.TODO(), "AzuriteTable", nil)
-handle(err)
+    client, err := svc.CreateTable(context.TODO(), "AzuriteTable", nil)
```
Comment (Duffney) [inline sdk/data/aztables/README.md:189]: Is `AzuriteTable` a typo? Is it supposed to be `AzureTable`?

Comment (seankane-msft) [reply sdk/data/aztables/README.md:189]: This sample is specifically for connecting to the Azurite emulator so I went with the name to match.

File: sdk/data/aztables/README.md [hunk 2]
```diff
@@ -39,11 +39,21 @@ az storage account show -n mystorageaccount -g MyResourceGroup --query "primaryE
...
-handle(err)
-serviceClient, err := aztables.NewServiceClientWithSharedKeyCredential("https://<myAccountName>.table.core.windows.net/", cred, nil)
-handle(err)
+```go
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/data/aztables"
+)
+
+func main() {
+	cred, err := aztables.NewSharedKeyCredential("<myAccountName>", "<myAccountKey>")
```
Comment (kimprice) [inline sdk/data/aztables/README.md:48]: Just a nitpick that sometimes you include the angle brackets for the string placeholder on this method and sometimes you leave it out. Maybe we should adopt a standard and try to be consistent throughout our READMEs/samples.

Comment (kimprice) [inline sdk/data/aztables/README.md:48]: Just a nitpick that sometimes you include the angle brackets for the string placeholder on this method and sometimes you leave it out. Maybe we should adopt a standard and try to be consistent throughout our READMEs/samples.

**Review by kimprice — APPROVED** [review]

**Pull request merged**